blog/Website/2015-03_05_FreshRSS_gets_a_new_website.md

```diff
@@ -0,0 +1,17 @@
+Title: FreshRSS gets a new website
+Date: 2015-03-05
+Comments_at: https://github.com/FreshRSS/freshrss.org/issues/16
+
+We have worked a lot to improve community processes since last weeks. Today we are opening a new section on the website: a blog!
+
+You were a lot to ask for an official English thread associated to a RSS feed to follow us (that was the last straw for FreshRSS!). It is now available so don't forget to add [our feed](http://freshrss.org/feeds/all.atom.xml) in your favourite aggregator.
+
+The paint is still wet but things should work smoothly since the blog is very basic.
+
+Under the hood, there is a static website [powered by Pelican](http://blog.getpelican.com/). The initial theme has been ported to this great static website generator tool and improved a bit (e.g. the header navbar). It is already available on GitHub ([FreshRSS/Freshican](https://github.com/FreshRSS/Freshican)) such as the content ([FreshRSS/freshrss.org](https://github.com/FreshRSS/freshrss.org)).
+
+As you may notice, there is no possibility to comment directly our articles: we want to avoid spam and moderation. Nevertheless you can meet us on GitHub, there will be a ticket associated to each article!
+
+Since we want to adopt a “community-friendly” process, you will have the possibility to modify and/or write articles in order to be published here. If you want to contribute to this blog, please fork [the project on GitHub](https://github.com/FreshRSS/freshrss.org), create a new branch, write your article and send a pull request. The only condition will be that you must speak about RSS/Atom aggregation :).
+
+To finish, stay in touch with us since there are still some great news to come in the next days!
```

Comment:
I would change `next days` by `next few days`. But apart from that, I think it is ok.
Let me know before I merge.